library-files-utilities/srfi/%3A%104/library-files-utilities.r6rs-lib

```diff
@@ -1,28 +1,8 @@
 #!r6rs  ;; Required by PLT Scheme.
 
-;; Copyright (c) 2010 Derick Eddington
-;;
-;; Permission is hereby granted, free of charge, to any person obtaining a
-;; copy of this software and associated documentation files (the "Software"),
-;; to deal in the Software without restriction, including without limitation
-;; the rights to use, copy, modify, merge, publish, distribute, sublicense,
-;; and/or sell copies of the Software, and to permit persons to whom the
-;; Software is furnished to do so, subject to the following conditions:
-;;
-;; The above copyright notice and this permission notice shall be included in
-;; all copies or substantial portions of the Software.
-;;
-;; Except as contained in this notice, the name(s) of the above copyright
-;; holders shall not be used in advertising or otherwise to promote the sale,
-;; use or other dealings in this Software without prior written authorization.
-;;
-;; THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-;; IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-;; FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
-;; THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-;; LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
-;; FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
-;; DEALINGS IN THE SOFTWARE.
+;;;; SPDX-FileCopyrightText: 2010 Derick Eddington
+;;;;
+;;;; SPDX-License-Identifier: MIT
```

Comment:
Alas, this has the same issue as SRFI 103.  It has the same author, too.

Reply:
We'll have to wait for their answer. If they do not reply I'll have to extract the license text and tag it with a custom SPDX license tag.

Reply:
Derick has been kind enough to reply by email to you and I that they accept to change the license from their MIT variant to plain MIT (which this change is doing); so this should now be fine!